### ISSUE TITLE
What are the supported Date and Time Formats

### ISSUE THREAD

**Issue by manishprasadthapliyal** [explosion/spaCy#2434]
I am using the following models in my application:

- en_core_web_sm

- xx_ent_wiki_sm
I wanted to know the supported Date and Time formats that default Spacy model can extract.

## Your Environment

* Operating System:Windows/ubuntu
* Python Version Used:3.6
* spaCy Version Used: 2.0.x
* Environment Information: anaconda

**Comment by ines** [comment]
See [my comment on StackOverflow](https://stackoverflow.com/a/50794950/6400719):

> The English models were trained on the OntoNotes 5 corpus, which supports the [more extensive label scheme](https://spacy.io/api/annotation#named-entities) including `DATE` and `TIME`. 
>
> The `xx_ent_wiki_sm` model was trained on a Wikipedia corpus with a [more limited](https://spacy.io/api/annotation#ner-wikipedia-scheme) label scheme and only recognises `PER`, `LOC`, `ORG` and `MISC` out of the box ([model details here](https://spacy.io/models/xx#xx_ent_wiki_sm)).
>
> When using the models to extract mentions of date and time, it's important to keep in mind that it's a statistical process – so the results you see will depend on the context and the data the models were trained on. Depending on the texts you're working with, you likely want to [update and fine-tune](https://spacy.io/usage/training#ner) the pre-trained models with more examples specific to your application, or try a rule-based approach instead. Also see [this thread](https://github.com/explosion/spaCy/issues/513#issuecomment-377273861) for more details on date and time parsing.
